cases/Lasher_Trapp/Lasher_Trapp_series_comparison.py

```diff
@@ -12,7 +12,8 @@
 # activate latex text rendering
 rc('text', usetex=True)
 
-Lasher_Trapp_vars = ["lwp", "rwp", "surf_precip", "acc_precip", "cl_nc"]
+##Lasher_Trapp_vars = ["lwp", "rwp", "surf_precip", "acc_precip", "cl_nc"]
+Lasher_Trapp_vars = ["RH_max", "cloud_top_height", "surf_precip", "acc_precip", "acc_vol_precip"]
```

Comment:
Instead of changing the Lasher_Trapp case, please add a new one called "ICMW2020_cumulus_congestus"